vim25/soap/client.go

```diff
@@ -477,6 +478,42 @@ func (c *Client) SetCertificate(cert tls.Certificate) {
 	t.TLSClientConfig.Certificates = []tls.Certificate{cert}
 }
 
+// UseServiceVersion sets Client.Version to the current version of the service endpoint via /sdk/vimServiceVersions.xml
+func (c *Client) UseServiceVersion(kind ...string) error {
+	ns := "vim"
+	if len(kind) != 0 {
+		ns = kind[0]
+	}
+
+	u := c.URL()
+	u.Path = path.Join("/sdk", ns+"ServiceVersions.xml")
```

Comment:
Note that before the move `"/sdk"` was `vim25.Path` constant, that was all we were using in the vim25 package for this method.